noisyatom/blogs/tests/test_models.py

```diff
@@ -0,0 +1,56 @@
+
+from django.test import TestCase
+from django.utils.text import slugify
+
+from blogs.models import Post
+
+
+class PostModelTest(TestCase):
+
+	def setUp(self):
+		self.post = Post.objects.create(title='New title test', slug='first-new-title-test-created')
+
+	def create_post(self, title='Post new blog'):
+		return Post.objects.create(title=title)
+
+	def test_create_title(self):
+		obj = Post.objects.get(slug='first-new-title-test-created')
+
+		self.assertEqual(obj.title, 'New title test')
+		self.assertTrue(obj.content == '')
+	
+	def test_create_slug(self):
+		title_one = 'The new title for slug'
+		title_two = 'The new title for example with more char'
+		'''
+		The test three is only to show that if we have two title is the same or duplicate will give an error
+		because we said in the slug unique=true 
+		'''
+		title_three = 'The new title for slug'
```

Comment:
Should the doc string not be at the start of the function or class?

Reply:
The reason of this doc string is about the third object created in the test, please have another look